src/resources/js/dialog.js

```diff
@@ -1,17 +1,17 @@
 var tribe = tribe || {};
 tribe.dialogs = tribe.dialogs || {};
-tribe.dialogs.dialogs = tribe.dialogs.dialogs || {};
+tribe.dialogs.dialogs = tribe.dialogs.dialogs || [];
 tribe.dialogs.events = tribe.dialogs.events || {};
 
-	( function ( obj ) {
+	( function( obj ) {
 		'use strict';
 
 		document.addEventListener(
 			'DOMContentLoaded',
-			function () {
-				tribe.dialogs.dialogs.forEach(function(dialog) {
+			function() {
+				tribe.dialogs.dialogs.forEach( function( dialog ) {
 					var objName     = 'dialog_obj_' + dialog.id;
-					window[objName] = new window.A11yDialog({
+					window[objName] = new window.A11yDialog( {
```

Comment:
we should also store that variable inside of the `dialogs` object stored on the `obj`.
Also spacing around `[]`